.github/actions/rust-toolchain/action.yml

```diff
@@ -0,0 +1,25 @@
+name: rust-toolchain
+inputs:
+  toolchain:
+    description: |
+      Rust toolchain name.
+      See https://rust-lang.github.io/rustup/concepts/toolchains.html#toolchain-specification
+    required: false
+  target:
+    description: Target triple to install for this toolchain
+    required: false
+  components:
+    description: Space-separated list of components to be additionally installed for a new toolchain
+    required: false
+
+runs:
+  using: composite
```

Comment:
Should this be `composite`? Only a single step is run :thinking:

Reply:
See https://docs.github.com/en/free-pro-team@latest/actions/creating-actions/metadata-syntax-for-github-actions#runs-for-composite-run-steps-actions
The alternatives are docker or node.